tabpfn_client/remote_tabpfn_classifier.py

```diff
@@ -35,26 +35,46 @@ def __init__(
         #  In the future version, these configs will be used to create per-user TabPFNClassifier,
         #    allowing the user to setup the desired TabPFNClassifier on the server.
         # config for tabpfn
-        self.model = model
+        # self.model = model
         self.device = device
         self.base_path = base_path
         self.model_string = model_string
         self.batch_size_inference = batch_size_inference
-        self.fp16_inference = fp16_inference
-        self.inference_mode = inference_mode
-        self.c = c
+        # self.fp16_inference = fp16_inference
+        # self.inference_mode = inference_mode
+        # self.c = c
         self.N_ensemble_configurations = N_ensemble_configurations
-        self.preprocess_transforms = preprocess_transforms
+        # self.preprocess_transforms = preprocess_transforms
         self.feature_shift_decoder = feature_shift_decoder
-        self.normalize_with_test = normalize_with_test
-        self.average_logits = average_logits
-        self.categorical_features = categorical_features
-        self.optimize_metric = optimize_metric
+        # self.normalize_with_test = normalize_with_test
+        # self.average_logits = average_logits
+        # self.categorical_features = categorical_features
+        # self.optimize_metric = optimize_metric
         self.seed = seed
-        self.transformer_predict_kwargs_init = transformer_predict_kwargs_init
+        # self.transformer_predict_kwargs_init = transformer_predict_kwargs_init
         self.multiclass_decoder = multiclass_decoder
 
         self.inference_handler = inference_handler
+        self.inference_handler.config_tabpfn(
```

Comment:
It would be good to allow users to change the settings later on, just like they can with he classical TabPFN. That means we should only when `predict_proba` is called create the config.

You could hand this config over to the inference_handler in predict_proba as well.